src/main/webapp/question.html

```diff
@@ -23,7 +23,7 @@
     <!-- Navbar -->
     <nav class="navbar navbar-expand-lg navbar-light bg-light">
       <!-- Logo -->
-      <a class="navbar-brand" href="#">
+      <a class="navbar-brand" href="index.html">
```

Comment:
shouldn't it be "/index.html" ?

Reply:
It's in the same folder as index.html, so it doesn't matter, but we could do it just to be safe. But if we do, we must do the same to signup.html.

Reply:
It works fine without the dash but whatever you guys think it's best.

Reply:
if it works it's ok